tests/test_ptc.py

```diff
@@ -923,6 +962,694 @@ def test_ptcFitBootstrap(self):
         for (fitType, order) in [('POLYNOMIAL', 2), ('POLYNOMIAL', 3), ('EXPAPPROXIMATION', None)]:
             self.ptcFitAndCheckPtc(fitType=fitType, order=order, doFitBootstrap=True)
 
+    def _getSampleMeanAndVar(self, dense=False, mode="normal"):
+        """Get sample mean/var vectors and ptcTurnoff from LSSTCam data.
+
+        The ptcTurnoff values here were obtained by looking at the data
+        and what the code does. These may be changed in the future if
+        we decide on a different way of computing the turnoff.
+
+        Parameters
+        ----------
+        dense : `bool`, optional
+            Return a dense PTC?
+        mode : `str`, optional
+            The ptc type, should be "normal", "upturn", or "dip".
+
+        Returns
+        -------
+        rawMeans : `np.ndarray`
+            Array of raw mean values.
+        rawVars : `np.ndarray`
+            Array of raw variance values.
+        ptcTurnoff : `float`
+            PTC turnoff determined from fitting by eye.
+        """
+        if dense and mode == "normal":
+            # Taken from dense run 13591, detector 94, amplifier C02
+            ptcTurnoff = 92239.4794
+            rawMeans = np.array([
```

Comment:
I have a slight preference for storing these as external PTC dataset files in `tests/data`, but it's probably not worth the effort on this ticket.

Reply:
Since we didn't have a test data directory I was too lazy to make one ...